api/NonDocumentTypeChildNode.json

```
@@ -119,7 +119,9 @@
               "version_added": "4"
             },
             "ie": {
-              "version_added": "9"
+              "version_added": "9",
+              "partial_implementation": true,
+              "notes": "Only implemented for <code>Element</code>, not for <code>CharacterData</code>. See <a href='#Polyfill_for_Internet_Explorer_9_and_Safari'>polyfill</a>."
```

Links in BCD need to use absolute URLs (because the data can be used other places than the mdn_url above)

```suggestion
              "notes": "Only implemented for <code>Element</code>, not for <code>CharacterData</code>. See <a href='https://developer.mozilla.org/docs/Web/API/NonDocumentTypeChildNode/nextElementSibling#Polyfill_for_Internet_Explorer_9_and_Safari'>polyfill</a>."
```